src/translation-assistant.js

```diff
@@ -398,6 +425,40 @@ function translateMath(math, lang) {
     return math;
 }
 
+/**
+ * Perform regex substitutions on translated math strings
+ * so that it matches math translations that we do in translateMath()
+ *
+ * @param {string} math A user-translated math expression.
+ * @param {string} lang The locale of the translation language.
+ * @returns {string} The translated math expression.
+ */
+function normalizeTranslatedMath(math, lang) {
+
+    // NOTE(danielhollas): Maybe we could apply this regardless of locales
```

Comment:
What about defining a new const for use by both `translateMath` and `normalizeTranslatedMath`?

For example, `const THOUSAND_SEPARATOR_AS_THIN_SPACE_LOCALES = [...];`

I also noticed that `hu` is missing in this list by is in the `translateMath` one?

As well, as it possible to add a couple of tests (pass, fail) for this new function? I know it's tested by extension, but it would be good to also have a couple of test cases to demonstrate functionality.

Reply:
Sure, that makes sense. Should I put it at the beginning of the file or just before these two functions?

Ad tests: Do you mean I should specifically call this function in the test suite?
(translateMath is not tested in this way...)

Reply:
normalizeTranslatedMath is tested specifically in these 3 tests:
https://github.com/Khan/translation-assistant/pull/11/files#diff-dfcb0e876142d98f825c4a224127aed8R591
Should I add a test calling this function specifically?

Reply:
> Should I put it at the beginning of the file or just before these two functions?

I think at the top of the file with the other constants would work.

> Do you mean I should specifically call this function in the test suite?

Yes. If you're going to be using `THOUSAND_SEPARATOR_AS_THIN_SPACE_LOCALES` here, you could loop through the locales to verify expected output, or pick just one from the list to test? And for a negative test case input `en` or a fake locale.

> (translateMath is not tested in this way...)

I think it's getting sufficiently complex, and especially now with specific ordering of regexes (ie. Decimal comma), that it may not be a bad idea to add some "smoke tests" to ensure that future changes do not brake anything.

Reply:
> I think at the top of the file with the other constants would work.

In the end, I opted to put it right before the definition of translateMath. It seems better to have all locale lists near each other. Does that make sense?

I tried adding tests for both translateMath and normalizeTranslatedMath. Please, have a look.

Reply:
Makes sense!